modules/gapi/include/opencv2/gapi/s11n.hpp

```diff
@@ -209,6 +286,23 @@ static GCompileArg exec(cv::gapi::s11n::IIStream& is, const std::string& tag) {
 }
 };
 
+template<typename T> struct deserialize_runarg;
```

Comment:
what is about to name structure above : ``deserialize_carg`` and this one ``deserialize_rarg``?

It is only a suggsetion, no actions required if you don't think it is reasonable

Reply:
`carg` and `rarg` are harder to understand